.github/workflows/scripts/pr_workflow.py

```diff
@@ -276,12 +301,20 @@ def issue_comment_label_actions(
             LOGGER.info(f"Processing labels: {labels}")
             for label, action in labels.items():
                 if label == LGTM_LABEL_STR:
-                    label = f"{LGTM_BY_LABEL_PREFIX}{self.user_login}"
+                    if self.user_login == self.pr.user.login:
+                        LOGGER.info("PR submitter cannot approve for their own PR")
```

Comment:
```suggestion
                        LOGGER.error("PR submitter cannot approve for their own PR")
```

Reply:
See first comment, IMHO this should not be treated as an error, it is information but the action itself ran fine

Reply:
but this is an error message; it is not about the GH action it is about a user not being allowed to approve

Reply:
I disagree, it is informational, the user is not allowed to kick off our internal automation but there's no failure in that. If someone from a different community commented on our PRs I wouldn't call that an error, we just don't want to grant them permission to perform the automated steps of the action.

Reply:
See prior comment, completely disagree